tea-config-generator/README.md

```diff
@@ -0,0 +1,66 @@
+# CMR TEA Configuration Generator
+A lambda project to generate [Thin Egress App][teacode] (TEA) configuration files using [Serverless][sls]
+
+## Overrview
+A set of lambda functions to run under the CMR domain name which generate a TEA configuration.
+
+See: [Cumulus thin egress app][tea]
+
+## Usage
+
+The [run.sh](run.sh) command will execute many of the stages of the software needed
+get the application started. To see the options supported, run `./run.sh -h`. To start the application, run in one terminal window: `./run.sh -o`. This will install the lambda functions and start the offline server. In another terminal window try
+the following commands:
+
+    curl 'http://localhost:3000/dev/configuration/tea/capabilities'
+    curl -I 'http://localhost:3000/dev/configuration/tea/status'
+    curl -H 'Cmr-Token: uat-token-value' 'http://localhost:3000/dev/configuration/tea/provider/POCLOUD'
```

Comment:
Is this just an EDL bearer token? Can this header be `Authorization` instead?

Reply:
good catch, I renamed that and missed the readme